lib/link_checker/uri_checker/http_checker.rb

```diff
@@ -291,6 +291,7 @@ def additional_connection_headers
     end
 
     def use_google_safebrowsing?
+      return false if gov_uk_uri?
```

Comment:
Mentioned in Slack, but I think there are some URLs we would want to still check such as attachments. Wonder if we can have some sort of whitelist or blacklist.

I do agree with the sentiment though that it's more important that we check external URLs through the Google SafeBrowsing API.

Reply:
@thomasleese I've included a check for if it is a "/government/upload" but this should still reduce the number of requests made significantly